모의고사.js

```diff
@@ -0,0 +1,23 @@
+function solution(answers) {
+    const answer = [];
+    
+
+    let pik1 = [1,2,3,4,5];
+    let pik2 = [2,1,2,3,2,4,2,5];
+    let pik3 = [3,3,1,1,2,2,4,4,5,5];
+    
+
+    let num1 = answers.filter ( (a, i) => a === pik1[i%5] ).length;
+    let num2 = answers.filter ( (a, i) => a === pik2[i%8] ).length;
+    let num3 = answers.filter ( (a, i) => a === pik3[i%10] ).length;
+    
+
+    let max = Math.max (num1, num2, num3);
```

Comment:
1. 여기서 변수들은 전부 변하지 않기 때문에 const로 선언
2. 공백 신경쓰기
3. 5 , 8, 10 이렇게 보다는 pik1.length 이런식이 더 좋습니다
4. filter를 이용한 풀이는 굿 👍 a라는 변수명 보다 el가 더 나을 듯
```suggestion
    const num1 = answers.filter((el, i) => el === pik1[i % pik1.length]).length;
    const num2 = answers.filter((el, i) => el === pik2[i % pik2.length]).length;
    const num3 = answers.filter((el, i) => el === pik3[i % pik3.length]).length;
    
    const max = Math.max(num1, num2, num3);
```